app/controllers/users_controller.rb

```diff
@@ -20,4 +21,26 @@ def user_params
     params.require(:user).permit(:first_name, :last_name, :email, :password)
   end
 
+  def user_already_exists?
+    User.find_by_email(user_params[:email]) ? true : false
+  end
+
+  def try_again
+    flash[:success] = "Email already in use"
```

Comment:
Same flash thing here!

Reply:
I like this try_again method though, nice refactor

Reply:
Done. Thanks Haydon